app/src/main/java/com/iron/espresso/presentation/home/study/list/LocationFragment.kt

```diff
@@ -72,8 +82,12 @@ class LocationFragment : BaseFragment<FragmentLocationBinding>(R.layout.fragment
     companion object {
         private const val SORT_LENGTH = "length"
         private const val OPTION = "distance"
-        fun newInstance() =
-            LocationFragment()
+        fun newInstance(category: String) =
+            LocationFragment().apply {
+                arguments = Bundle().apply {
+                    putString(StudyListActivity.STUDY_CATEGORY, category)
```

Comment:
```suggestion
                    putString(STUDY_CATEGORY, category)
```
외부의 key값을 참조하는게 어색한 것 같아요. 내부에 private 으로 정의해주는게 적합할 것 같아요.
자기가 주입하고, 자기가 꺼내쓰는데 key만 외부에서 가져다 쓴다? 
인 상태라서

Reply:
너무 여기저기서 쓴다 싶으면 StudyListActivity의 companion object에 두지말고
```
object Study {
    const val STUDY_CATEGORY = "study_category"
} 
```
이런 파일을 공용으로 만드는 것도 방법이 될 수 있을 듯 합니다.

Reply:
저번에 한 행동을 그래도 실천해 버렸네요..ㅠ 내부에 정의 하는게 저도 적합해 보이네유 바로 수정하겠습니다
0746216